notebooks/224-vision-image-colorization/224-vision-image-colorization.ipynb

```diff
@@ -0,0 +1,511 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1662ee2c-fdcd-4b68-a65d-3b03a9e6aadf",
+   "metadata": {},
+   "source": [
+    "### Image Colorization with OpenVino\n",
+    "\n",
+    "This notebook demonstrates how to colorize images with OpenVINO using the Colorization model [colorization-v2](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/colorization-v2/README.md) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md) based on the paper [Colorful Image Colorization](https://arxiv.org/abs/1603.08511).\n",
```

Comment:
Could you include colorization-siggraph. Could you check if we can use the same notebook for the two models? colorization-v2 and colorization-siggraph.

Reply:
@paularamo I have included the notebook to handle colorization-siggraph in the same workflow, by providing option to modify the name of model to be downloaded from model zoo.